### ISSUE TITLE
Enabled plugins not persistent after program exits (Trac #62)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#66]
If a plugin is enabled and user exits the program, that plugin should start in an enabled state the next time that the program starts. A list of enabled plugins should be stored in the config file.

Migrated from http://microfluidics.utoronto.ca/ticket/62
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "If a plugin is enabled and user exits the program, that plugin should start in an enabled state the next time that the program starts. A list of enabled plugins should be stored in the config file.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Enabled plugins not persistent after program exits", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-02T01:55:13", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120309T04:22:01: 
* christian@fobel.net commented:

> commit 32d286393d3bdfa0f902b62e2d17353b8341c4c6
> Author: Christian Fobel <christian@fobel.net>
> Date:   Thu Mar 8 23:17:27 2012 -0500
> 
>     Update and save enabled plugins list to config...
>     
>     Update and save enabled plugins list to config.ini
>     Fixes #66.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "plugins" to "microdrop"
